apps/extension/package.json

```diff
@@ -5,13 +5,15 @@
   "description": "chrome-extension",
   "type": "module",
   "scripts": {
-    "dev": "webpack --watch --mode=development -d inline-source-map",
+    "dev": "NODE_ENV=testnet pnpm bundle --watch --mode=development -d inline-source-map",
+    "build": "NODE_ENV=mainnet pnpm bundle",
     "clean": "rm -rfv dist bin",
-    "build": "webpack",
+    "bundle": "NODE_OPTIONS=\"--import=./src/utils/webpack-register.js\" webpack",
```

Comment:
Now that webpack config is typescript, we need to use a special kind of register so it can the cli can load it